src/Compilers/CSharp/Portable/Lowering/NullCheckRewriter.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.CodeGen;
+using Microsoft.CodeAnalysis.CSharp.Symbols;
+using System.Collections.Immutable;
+using Microsoft.CodeAnalysis.PooledObjects;
+using System.Linq;
+
+namespace Microsoft.CodeAnalysis.CSharp.Lowering
```

Comment:
>Microsoft.CodeAnalysis.CSharp.Lowering [](start = 10, length = 38)

We don't have a `Lowering` namespace currently. Consider changing this to `Microsoft.CodeAnalysis.CSharp`.